features/ubuntu_pro.feature

```diff
@@ -664,6 +668,17 @@ Feature: Command behaviour when auto-attached in an ubuntu PRO image
         """
         Skipping auto-attach and deferring to cloud-init to setup and configure auto-attach
         """
+        When I run `cloud-init status` with sudo
+        Then stdout matches regexp:
+        """
+        status: done
+        """
+        When I run `pro status --wait` with sudo
+        And I run `pro security-status --format json` with sudo
+        Then stdout matches regexp:
+        """
+        "attached": false
+        """
```

Comment:
I think I am missing something here. Why are we repeating this check ? Since we do the same thing on lines 656-661

Reply:
This is not needed, it is a relic of the previous PR attempt, where we had a multi-snapshot setup, and we were exercising this check in both snapshots.

I will remove it and push when AWS images will be ready.